spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/catalog/BaseCatalog.java

```diff
@@ -195,26 +197,40 @@ public Table createTable(
                   partitionings,
                   distributionAndSortOrdersInfo.getDistribution(),
                   distributionAndSortOrdersInfo.getSortOrders());
+      Table sparkTable = sparkCatalog.loadTable(ident);
```

Comment:
could sparkTable be loaded lazily?

Reply:
it seems can't.
Because we need to call the `super` method in the constructor of the `SparkIcebergTable` or `SparkHiveTable`

Reply:
got it